main/src/cgeo/geocaching/ui/dialog/CoordinatesCalculateDialog.java

```diff
@@ -0,0 +1,1096 @@
+                setCoordValue(gp.getLonMinFrac(), bLonPnt_10000, bLonPnt_01000, bLonPnt_00100);
+                break;
+
+            case Sec: // DDD° MM' SS.SSS"
+                setVisible(coordTable, View.VISIBLE);
+                setVisible(PlainFormat, View.GONE);
+
+                for (final View view : minButtons) {
+                    view.setVisibility(View.VISIBLE);
+                }
+                for (final View view : secButtons) {
+                    view.setVisibility(View.VISIBLE);
+                }
+                for (final EditButton button : pointLowButtons) {
+                    button.setVisibility(View.GONE);
+                }
+                for (final TextView view : lastUnits) {
+                    view.setText(SYMBOL_SEC);
+                }
+
+                setCoordValue(gp.getLatSecFrac(), bLatPnt_10000, bLatPnt_01000, bLatPnt_00100);
+                setCoordValue(gp.getLonSecFrac(), bLonPnt_10000, bLonPnt_01000, bLonPnt_00100);
+                break;
+        }
+    }
+
+    /**
+     * Find if a variable exists in the supplied list with the given name.
+     * @param name: The name to search for.
+     * @return: The first occurrence of the variable if it can found, 'null' otherwise.
+     */
+    private CalculatorVariable findVariable(final char name, final List<CalculatorVariable> list) {
+        for (final CalculatorVariable equ : list) {
+            if (equ.getName() == name) {
+                return equ;
+            }
+        }
+
+        return null;
+    }
+
+    // Resort the equations into the order in which they first appear in the 'buttons' or 'plain-text' fields as appropriate.
+    // This method is also responsible for adding and removing equations from the associated list as appropriate.
+    private void resortEquations() {
+        final List<CalculatorVariable> resortedEquations = new ArrayList<>();
+
+        String chars = "";
+
+        if (currentFormat == Settings.CoordInputFormatEnum.Plain) {
+            final String lat = ePlainLat.getText().toString();
+            final String lon = ePlainLon.getText().toString();
+            char first;
+
+            if (lat.length() > 0) {
+                first = lat.charAt(0);
+                if (first == 'N' || first == 'S') {
+                    chars = chars.concat(lat.substring(1));
+                } else {
+                    chars = chars.concat(lat);
+                }
+            }
+
+            if (lon.length() > 0) {
+                first = lon.charAt(0);
+                if (first == 'E' || first == 'W') {
+                    chars = chars.concat(lon.substring(1));
+                } else {
+                    chars = chars.concat(lon);
+                }
+            }
+        } else {
+            for (final EditButton b : coordButtons) {
+                chars = chars.concat(String.valueOf(b.getLabel()));
+            }
+        }
+
+        for (int i = 0; i<chars.length(); i++) {

```

Comment:
"Probably the missing spaces"??

I would normally write this as: `for (int i=0 ; i<chars.length() ; i++)` however Codacy wants spaces around the '='.